cajita/src/Cajita_FastFourierTransform.hpp

```diff
@@ -163,25 +164,26 @@ class FastFourierTransform
       \param params Parameters for the 3D FFT.
     */
     FastFourierTransform( const ArrayLayout<EntityType, MeshType> &layout,
-                          const FastFourierTransformParams &params )
-        : _fft( layout.localGrid()->globalGrid().comm() )
+                        //   const FastFourierTransformParams &params )
+                          const heffte::plan_options &params )
+        // : _fft( layout.localGrid()->globalGrid().comm() )
     {
         if ( 1 != layout.dofsPerEntity() )
             throw std::logic_error(
                 "Only 1 complex value per entity allowed in FFT" );
 
         // Set the memory type. For now we will just do FFTs on the host until
         // we find the HEFFTE GPU memory bug.
-        _fft.mem_type = HEFFTE_MEM_CPU;
+        // _fft.mem_type = HEFFTE_MEM_CPU;  // * Memory automatically defined from backend type
```

Comment:
This implies that the `HeffteMemoryTraits` (lines 31-52) don’t seem to be needed anymore. So I think we should go ahead and remove them.